csv/src/main/java/com/fasterxml/jackson/dataformat/csv/impl/TextBuffer.java

```diff
@@ -313,12 +313,24 @@ public BigDecimal contentsAsDecimal()
      * Convenience method for converting contents of the buffer
      * into a Double value.
      */
-    public double contentsAsDouble()
-        throws NumberFormatException
-    {
+    public double contentsAsDouble() throws NumberFormatException {
```

Comment:
Mark as `@Depracated`, easier to find calls.

Reply:
done